app/services/spaces/aggregate_star_rating_service.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Spaces
+  class AggregateStarRatingService < ApplicationService
+    def initialize(space:)
+      @space = space
+      super()
+    end
+
+    def call
+      star_rating = space.reviews.all.sum(&:star_rating)
+
+      space.update!(star_rating: star_rating / space.reviews.count)
```

Comment:
This will divide by zero if there are no reviews, no?

Could be triggered if the last review is deleted, and we want to recalculate the star rating.

Reply:
Yep good point